src/Raven.Server/Rachis/RachisLogHistory.cs

```diff
@@ -49,6 +50,27 @@ private enum HistoryStatus : byte
             Committed
         }
 
+        private static long _lastTicks;
+        private static long GetUniqueTicks()
```

Comment:
Let's avoid using `static` here, I want to be able to run this concurrently on two Rachis instances without issue.
We don't need multi threading support, since we can assume that this is in a write transaction.
Let's accept the transaction as a parameter here and check that we have a write transaction to avoid future issues.